data/_models/shared/Contacts.csv

```diff
@@ -1,16 +1,17 @@
-tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Contacts,,,,,,,,,,,vcard:Individual,Listing of contact persons per resource,"DataCatalogueFlat,EMA",
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible,computed
+Contacts,,,,,,,,,,,"vcard:Individual, foaf:Agent",Listing of contact persons per resource,"DataCatalogueFlat,EMA",
```

Comment:
lose the space here. And according to dcat-ap 3.0 and health-ri we should use vcard:Kind https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification#Dataset with just the properties